snuba/utils/metrics/backends/datadog.py

```diff
@@ -75,3 +75,19 @@ def timing(
             tags=self.__normalize_tags(tags),
             sample_rate=self.__sample_rates.get(name, 1.0),
         )
+
+    def events(
+        self,
+        title: str,
+        text: str,
+        alert_type: str,
+        priority: str,
+        tags: Optional[Tags] = None,
+    ) -> None:
+        self.__client.event(
+            title=title,
+            text=text,
+            alert_type=alert_type,
+            tags=self.__normalize_tags(tags),
+            priority=priority,
+        )
```

Comment:
The answer might be yes, but can you check that the datadog client version we use does actually support sending events?

Reply:
yes. confirmed